source/GameInterface/Services/ItemRosters/Messages/ItemRosterUpdate.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Common.Logging.Attributes;
+using Common.Messaging;
+
+namespace GameInterface.Services.ItemRosters.Messages
+{
+    /// <summary>
+    /// Called when an item roster is updated.
+    /// </summary>
+    [BatchLogMessage]
+    public class ItemRosterUpdate : IEvent
```

Comment:
Create 2 messages, `ItemRosterUpdated : IEvent` and `ItemRosterUpdate : ICommand`

ItemRosterUpdated will be called by the prefix and ItemRosterUpdate will be used by the client to update the ItemRoster